src/SupabaseStorageClient.ts

```diff
@@ -15,3 +15,8 @@ export class SupabaseStorageClient extends StorageBucketApi {
     return new StorageFileApi(this.url, this.headers, id, this.fetch)
   }
 }
+
+//! Alias `StorageClient` as `SupabaseStorageClient` for backwards compatibility.
```

Comment:
Why are you using `//!`? As far as I understand it's used so that some minifiers might not strip the comment out, which shouldn't be applicable here?

Reply:
Force of habit. I have a VSCode extensions called 'BetterComments' which highlights comments based on some format rules.

Reply:
Ah, many people do use that extension, I've no issue with leaving that in.